examples/simulation/plot_simulate_evoked_data.py

```diff
@@ -26,22 +24,22 @@
 # Load real data as templates
 data_path = sample.data_path()
 
-raw = Raw(data_path + '/MEG/sample/sample_audvis_raw.fif')
-proj = read_proj(data_path + '/MEG/sample/sample_audvis_ecg_proj.fif')
+raw = mne.io.read_raw_fif(data_path + '/MEG/sample/sample_audvis_raw.fif')
```

Comment:
I presume that our examples don't work for windows machine b/c of the `/`. should we be more systematic and use `op.join` which should take care of the folder structure given the os? this doesn't need to be addressed in this PR